src/scripts/generate-unary-ops.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+import sys
+import os
+import settings
+import random
+path = os.path.join(settings.TEMPLATE_DIR)
+sys.path.append(path)
+import validate
+import messages
+from constants import _constants
+from parse_args import parse_args
+from generator_base import Generator
+from utils import write_file
+
+_GENERATED_FILENAME = '''unary-ops-{NUMBER}.js''' # The file name of the generated tests
+_UNARY_OPERATORS = ["+", "-", "++", "--", "!", "~", "typeof", "delete", "void"] # The unary operators used in this script
+_FALSE_VALUES_COUNT = 5 #The number of false values generated for each tests
+_RELATION_OPERATORS = ["<", ">", "<=", ">=", "==", "!="] #Relation operators used while genrating tests for ! operator
+
+class Unary_Operator_Generator(Generator):
+    def create_operation(self, op, count, i): # Creates the operation's string
+        if op in ["+", "-", "++", "--", "~", "!"]:
```

Comment:
This array is the subarray of unary operators. Please regroup them, to declare the elements only once.